frontend/src/services/AuthService.ts

```diff
@@ -34,7 +34,15 @@ async function login() {
         https://docs.amplify.aws/lib/auth/social/q/platform/js/
         https://docs.amplify.aws/lib/auth/advanced/q/platform/js/#identity-pool-federation
     */
-    Auth.federatedSignIn();
+    // Auth.federatedSignIn();
+
+    const env = JSON.parse(window.localStorage.getItem('env_data') || '');
```

Comment:
The EnvironmentSettings class should be used instead as that abstracts accessing environment-specific data.